rasa/core/training/interactive.py

```diff
@@ -579,6 +579,18 @@ def _slot_history(tracker_dump: Dict[Text, Any]) -> List[Text]:
     return slot_strings
 
 
+async def _retry_on_error(func: Callable, export_path: Text, *args, **kwargs):
+    while True:
+        try:
+            return await func(export_path, *args, **kwargs)
+        except OSError as e:
+            answer = questionary.confirm(
+                f"Failed to export {export_path}: {e}. Please fix issues and retry."
+            ).ask()
+            if not answer:
+                return  # raise Abort()
```

Comment:
In that case I'd raise the original error instead of failing silently.
```suggestion
                raise e
```